vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -226,3 +225,11 @@ Often it is only a matter of individual variables that are supposed to be sorted
 ```{r relocateVariable}
 dat11 <- relocateVariable(dat11, var = "age_FDZ", after = "age")
 ```
+
+
+## Exporting the data
```

Comment:
LGTM, thanks for spotting that this is missing from the vignette!

Nits:
Could you add the correct file extensions to the filePaths? For instance "filePath.sav"?
Maybe the wording "Once the data record has been prepared" is a bit ambigouos. How about:
"Once the data set has been cleaned, ..."

Reply:
Good idea, thanks for your input!